hicommonwealth/commonwealth
# 6819 growl wrong url

## Link to Issue
Closes: #6819 

## Description of Changes
- Added verbosity to the `onPress`

## "How We Fixed It"

- Added `(window.location.href = `${window.location.protocol}//${window.location.host}/createCommunity`)` to the `onPress` prop to ensure correct navigation.

## Test Plan
- Go to a community
- Press 'Create community with stake"
- ensure navigation is correct
